js/actions/appActions.js

```diff
@@ -1981,6 +1981,13 @@ const appActions = {
     })
   },
 
+  onFuzzing: function (newStamp) {
```

Comment:
Following other comment, maybe this should be `onLedgerFuzzing`

Reply:
done